modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/shibari_worn.dm

```diff
@@ -0,0 +1,333 @@
+
+
+#define SHIBARI_TIGHTNESS_LOW (1<<0)
+#define SHIBARI_TIGHTNESS_MED (1<<1)
+#define SHIBARI_TIGHTNESS_HIGH (1<<2)
+
+/obj/item/clothing/under/shibari
+	strip_delay = 100
+	can_adjust = FALSE
+	body_parts_covered = NONE
+	mutant_variants = STYLE_DIGITIGRADE|STYLE_TAUR_ALL
+	item_flags = DROPDEL
+	greyscale_colors = "#bd8fcf"
+
+	///Tightness of the ropes can be low, medium and hard. This var works as multiplier for arousal and pleasure recieved while wearing this item
+	var/tightness = SHIBARI_TIGHTNESS_LOW
+	///Rope amount yielded from this apparel
+	var/rope_amount = 1
+
+	///should this clothing item use the emissive system
+	var/glow = FALSE
+
+/obj/item/clothing/under/shibari/update_overlays()
+	. = ..()
+	if(glow)
+		. += emissive_appearance(icon, icon_state, alpha = alpha)
+
+/obj/item/clothing/under/shibari/worn_overlays(mutable_appearance/standing, isinhands, icon_file)
+	. = ..()
+	if(glow)
+		. += emissive_appearance(standing.icon, standing.icon_state, alpha = standing.alpha)
+
+/obj/item/clothing/under/shibari/Destroy(force)
+	STOP_PROCESSING(SSobj, src)
+
+	for(var/obj/item in contents)
+		item.forceMove(get_turf(src))
+	if(ishuman(loc))
```

Comment:
early return

Reply:
on destroy()? seriously

Reply:
Yep, you can just early return ..()

Reply:
yep, when I say early return, it doesn't mean you can't do `return ..()`

Reply:
why is skipping a few line of bytecodes so fucking important for an item like this

Reply:
Because early returns are the code standard that should ideally be enforced everywhere unless there's good reason (read: loss of functionality) otherwise.

Reply:
And because early returns makes your code less of a pain to read.